content/docs/userspace/dist/docket.md

```diff
@@ -0,0 +1,296 @@
++++
+title = "Docket File"
+weight = 3
+template = "doc.html"
++++
+
+The docket file sets various options for desks with a tile and (usually) a browser-based front-end of some kind. Mainly it configures the appearance of an app's tile, the source of its [glob](/docs/userspace/dist/glob), and some additional metadata.
+
+The docket file is read by the `%docket` agent when a desk is `|install`ed. The `%docket` agent will fetch the glob if applicable and create the tile as specified on the homescreen. If the desk is published with `:treaty|publish`, the information specified in the docket file will also be displayed for others who are browsing apps to install on your ship.
+
+The docket file is _optional_ in the general case. If it is omitted, however, the app cannot have a tile in the homescreen, nor can it be published with the `%treaty` agent, so others will not be able to browse for it from their homescreens.
+
+The docket file must be named `desk.docket-0`. The `%docket` `mark` is versioned to facilitate changes down the line, so the `-0` suffix may be incremented in the future.
+
+The file must contain a `hoon` list with a series of clauses. The clauses are defined in `/sur/docket.hoon` as:
+
+```hoon
++$  clause
+  $%  [%title title=@t]
+      [%info info=@t]
+      [%color color=@ux]
+      [%glob-http url=cord hash=@uvH]
+      [%glob-ames =ship hash=@uvH]
+      [%image =url]
+      [%site =path]
+      [%base base=term]
+      [%version =version]
+      [%website website=url]
+      [%license license=cord]
+  ==
+```
+
+The `%image` clause is optional. It is mandatory to have exactly one of either `%site`, `%glob-http` or `%glob-ames`. All other clauses are mandatory.
+
+Here's what a typical docket file might look like:
+
+```hoon
+:~
+  title+'Foo'
+  info+'An app that does a thing.'
+  color+0xf9.8e40
+  glob-ames+[~zod 0v0]
+  image+'https://example.com/tile.svg'
+  base+'foo'
+  version+[0 0 1]
+  license+'MIT'
+  website+'https://example.com'
+==
+```
+
+Details of each clause and their purpose are described below.
+
+---
+
+## `%title`
+
+_required_
+
+The `%title` field specifies the name of the app. The title will be the name shown on the app's tile, as well as the name of the app when others search for it.
+
+#### Type
+
+```hoon
+[%title title=@t]
+```
+
+#### Example
+
+```hoon
+title+'Bitcoin'
+```
+
+---
+
+## `%info`
+
+_required_
+
+The `%info` field is a brief summary of what the app does. It will be shown as the subtitle in _App Info_.
+
+#### Type
+
+```hoon
+[%info info=@t]
+```
+
+#### Example
+
+```hoon
+info+'A Bitcoin Wallet that lets you send and receive Bitcoin directly to and from other Urbit users'
+```
+
+---
+
+## `%color`
+
+_required_
+
+The `%color` field specifies the color of the app tile in hexadecimal `RRGGBB` color-code format.
```

Comment:
Actually, if it's black, or there's no red or w/e, this doesn't hold. Might be more clear to say it's the `@ux`-formatted hex value of the color. ie `0xff` for pure blue.